### PR TITLE
update core bundle documentation for 2.0

### PR DESCRIPTION
**Pull request by dbu**
part of #787

### CODE REVIEW

File: bundles/core/configuration.rst [hunk 1]
```diff
@@ -144,22 +138,28 @@ Enabling this setting will also automatically enable the equivalent setting in t
...
 """"""""""""""""""""
 
 **type**: ``string`` **default**: ``doctrine_phpcr``
 
-Enabling this setting will also automatically enable the equivalent settings in the following Bundles:
+The doctrine registry from which to get the document manager. This setting
+only needs to be changed when configuring multiple manager registries.
+
+This setting is propagated as default value to all CMF bundles that support
+this setting:
```
Comment (dbu) [inline bundles/core/configuration.rst]: i am rather surprised why the registry service name can be configured on search and simple bundle but not on any other. but as we already released them, i guess we have to leave that as is for now.

File: bundles/sonata_phpcr_admin_integration/core.rst [hunk 2]
```diff
@@ -1,2 +1,221 @@
 CoreBundle
 ==========
+
```
Comment (ElectricMaxxx) [inline bundles/sonata_phpcr_admin_integration/core.rst]: please add an hint, that this repository isn't stable atm - even when it gets stable next week.

File: bundles/sonata_phpcr_admin_integration/core.rst [hunk 3]
```diff
@@ -1,2 +1,221 @@
...
+Defines which form group the fields from this extension will appear in within
+the Sonata Admin edit interface.
+
+``form_tab``
+~~~~~~~~~~~~
+
+**type**: ``string`` **default**: as in above example.
+
+Defines which form tab the fields from this extension will appear in within
```
Comment (ElectricMaxxx) [inline bundles/sonata_phpcr_admin_integration/core.rst]: Not only, they also are passed into the translation so it is not a key only, it visible translated, so a non default key needs to be translated too.

File: bundles/core/configuration.rst [hunk 4]
```diff
@@ -125,7 +119,8 @@ is the following configuration:
 
 .. include:: ../_partials/persistence_phpcr_enabled.rst.inc
 
-Enabling this setting will also automatically enable the equivalent setting in the following Bundles:
+This setting is propagated as default value to all CMF bundles that support
```
Comment (dbu) [inline bundles/core/configuration.rst]: right. added a note for that in the introduction (as this note is repeated for each setting that can be forwarded)

File: bundles/sonata_phpcr_admin_integration/core.rst (hunk 2 above)
Comment (dbu) [inline bundles/sonata_phpcr_admin_integration/core.rst]: adding that hint to the introduction on the bundle

File: bundles/sonata_phpcr_admin_integration/core.rst (hunk 3 above)
Comment (dbu) [inline bundles/sonata_phpcr_admin_integration/core.rst]: good point, added that

**Pull request merged**